WordPress/Sniffs/WP/I18nSniff.php

```diff
@@ -0,0 +1,85 @@
+<?php
+/**
+ * WordPress_Sniffs_WP_I18nSniff
+ *
+ * Makes sure internationalization functions are used properly
+ *
+ * @category  PHP
+ * @package   PHP_CodeSniffer
+ * @author    Shady Sharaf <shady@x-team.com>
+ */
+class WordPress_Sniffs_WP_I18nSniff implements PHP_CodeSniffer_Sniff {
+
+	public $i18n_functions = array(
+		'translate',
+		'translate_with_gettext_context',
```

Comment:
translate() and translate_with_gettext_context() are low-API functions, they shouldn't be used by plugins IMO.